src/subdomains/generic/user/models/auth/auth-lnurl.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Controller, Get, Post, Query, Req } from '@nestjs/common';
+import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { Request } from 'express';
+import { RealIP } from 'nestjs-real-ip';
+import { AuthLnUrlService } from './auth-lnurl.service';
+import { AuthLnurlResponseDto, AuthLnurlSignupDto } from './dto/auth-lnurl.dto';
+
+@ApiTags('LNURL')
+@Controller('auth')
+export class AuthLnurlController {
+  constructor(private readonly lnUrlService: AuthLnUrlService) {}
+
+  @Post('lnurla')
+  async getLnurlAuth(): Promise<string> {
+    return this.lnUrlService.createLoginLnurl();
+  }
```

Comment:
Plain string responses are annoying in a client normally handling JSON. Additionally the client also needs the k1, to be able to fetch the result. So maybe return something like:
```ts
{
   k1: string,
   lnurl: string
}
```